test/integration/auth.spec.ts

```diff
@@ -1164,29 +1164,41 @@ describe('admin.auth', () => {
       recaptchaConfig: {
         emailPasswordEnforcementState:  'AUDIT',
         managedRules: [{ endScore: 0.1, action: 'BLOCK' }],
+        useAccountDefender: true,
       },
     };
     const projectConfigOption2: UpdateProjectConfigRequest = {
       recaptchaConfig: {
         emailPasswordEnforcementState:  'OFF',
+        useAccountDefender: false,
+      },
+    };
+    const projectConfigOption3: UpdateProjectConfigRequest = {
+      recaptchaConfig: {
+        emailPasswordEnforcementState:  'OFF',
+        useAccountDefender: true,
       },
     };
     const expectedProjectConfig1: any = {
       recaptchaConfig: {
         emailPasswordEnforcementState:  'AUDIT',
         managedRules: [{ endScore: 0.1, action: 'BLOCK' }],
+        useAccountDefender: true,
       },
     };
     const expectedProjectConfig2: any = {
       recaptchaConfig: {
         emailPasswordEnforcementState:  'OFF',
         managedRules: [{ endScore: 0.1, action: 'BLOCK' }],
+        useAccountDefender: false,
       },
     };
 
     it('updateProjectConfig() should resolve with the updated project config', () => {
       return getAuth().projectConfigManager().updateProjectConfig(projectConfigOption1)
         .then((actualProjectConfig) => {
+          // keys are generated randomly.
```

Comment:
Where are the keys generated?

Reply:
this is the response we got from restAPI, and everytime turn on/off the value changes.